### PR TITLE
Remove pds for 17.0.1.2.1

### PR DESCRIPTION
**Pull request by kneckinator**
## **Why is this change needed?**
Add to OpenSPP 17.0.1.2.1 Batanes release

### CODE REVIEW

File: spp_change_request/i18n/ar.po [hunk 1]
```diff
@@ -619,11 +619,6 @@ msgid ""
 msgstr ""
 "خطأ في مراحل التحقق من الصحة. لا توجد مرحلة متاحة لتعيين هذا التحقق من الصحة."
 
-#. module: spp_change_request
```
Comment (kneckinator) [inline spp_change_request/i18n/ar.po:622]: @gonzalesedwin1123 Will the translations for `spp_dms_extra_documents` be generated automatically, or should this section be kept and changed to match the change in data/dms.xml?

File: spp_change_request/i18n/ckb.po [hunk 2]
```diff
@@ -602,11 +602,6 @@ msgid ""
 "Error in validation stages. No available stage to assign to this validation."
 msgstr ""
 
-#. module: spp_change_request
```
Comment (kneckinator) [inline spp_change_request/i18n/ckb.po:605]: @gonzalesedwin1123 Will the translations for `spp_dms_extra_documents` be generated automatically, or should this section be kept and changed to match the change in data/dms.xml?

File: spp_change_request/i18n/fr.po [hunk 3]
```diff
@@ -635,12 +635,6 @@ msgstr ""
 "Erreur dans les étapes de validation. Aucune étape disponible à affecter à "
 "cette validation."
 
-#. module: spp_change_request
```
Comment (kneckinator) [inline spp_change_request/i18n/fr.po:638]: @gonzalesedwin1123 Will the translations for `spp_dms_extra_documents` be generated automatically, or should this section be kept and changed to match the change in data/dms.xml?

File: spp_service_points/tests/test_service_point.py [hunk 4]
```diff
@@ -85,7 +85,7 @@ def setUpClass(cls):
     #     self.assertNotEqual(
     #         self.service_point_1.phone_sanitized,
     #         False,
-    #         "Service Point 1 having correct Iraq phone number format, should have phone sanitized!",
+    #         "Service Point 1 having correct phone number format, should have phone sanitized!",
```
Comment (kneckinator) [inline spp_service_points/tests/test_service_point.py]: @gonzalesedwin1123 I don't understand this assertion message, as the use of "correct" is confusing me.

File: spp_change_request/i18n/ar.po (hunk 1 above)
Comment (gonzalesedwin1123) [reply spp_change_request/i18n/ar.po:622]: It should be kept @kneckinator and later be updated to match the change in data/dms.xml

File: spp_change_request/i18n/ckb.po (hunk 2 above)
Comment (gonzalesedwin1123) [reply spp_change_request/i18n/ckb.po:605]: It should be kept @kneckinator and later be updated to match the change in data/dms.xml

File: spp_change_request/i18n/fr.po (hunk 3 above)
Comment (gonzalesedwin1123) [reply spp_change_request/i18n/fr.po:638]: It should be kept @kneckinator and later be updated to match the change in data/dms.xml

File: spp_service_points/tests/test_service_point.py (hunk 4 above)
Comment (gonzalesedwin1123) [reply spp_service_points/tests/test_service_point.py]: Yes, I agree. The sentence needs to be improved. I'll work on this.

Comment (gonzalesedwin1123) [reply spp_service_points/tests/test_service_point.py]: @kneckinator After reading the code I am now not sure of what to put in the assertion message. Should it be "Service Point 1 have a correct phone number format but should be sanitized"?

Comment (kneckinator) [reply spp_service_points/tests/test_service_point.py]: I have no idea, either. :)

Comment (kneckinator) [reply spp_service_points/tests/test_service_point.py]: Let's leave it for now.

**Pull request merged**